packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -32,6 +29,13 @@ for PACKAGE in ${PACKAGES[@]}; do
     # name (e.g. [branch-name]-api.tupaia.org -> specific-branch-api.tupaia.org)
     sed -i -e "s/\[branch-name\]/${BRANCH}/g" .env
 
+    if [[ "$BRANCH" == *-e2e ]]; then
+        # Update e2e environment variables
+        if [[ $PACKAGE == "meditrak-server" || $PACKAGE == "web-config-server" ]]; then
+            sed -i -E 's/^AGGREGATION_URL_PREFIX="?dev-"?$/AGGREGATION_URL_PREFIX=e2e-/g' .env
```

Comment:
Oh nice, this is where you do the manual switch! I thought it would be in the js code, but this makes way more sense as it is a concern of our infrastructure rather than the code 🎉

Reply:
I thought the same initially, then I was lead here during development!